pkg/nvlib/info/info.go

```diff
@@ -76,3 +82,56 @@ func (i *infolib) IsTegraSystem() (bool, string) {
 
 	return false, fmt.Sprintf("%v has no 'tegra' prefix", tegraFamilyFile)
 }
+
+// UsesNVGPUModule checks whether the nvgpu module is used.
+// This kernel module is used on Tegra-based systems when using the iGPU.
+// Since some of these systems also support NVML, we use the device name
+// reported by NVML to determine whether the system is an iGPU system.
+//
+// Devices that use the nvgpu module have their device names as:
+//
+//	GPU 0: Orin (nvgpu) (UUID: 54d0709b-558d-5a59-9c65-0c5fc14a21a4)
+//
+// This function returns true if ALL devices use the nvgpu module.
+func (i *infolib) UsesNVGPUModule() (uses bool, reason string) {
```

Comment:
Maybe call it `OnlyUsesNVGPUModule` (or something else to indicate that it only returns true if *all*. GPUs are managed by this module).

Reply:
Yes, that's fair. Let me update.